indico/htdocs/js/indico/jquery/rulelistwidget.js

```diff
@@ -0,0 +1,213 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    'use strict';
+
+    function _listValue(option) {
+        if (option === '') {
+            return [];
+        } else {
+            var parsedValue = parseInt(option, 10);
+            return [isNaN(parsedValue) ? option : parsedValue];
+        }
+    }
+
+    function _optionValue(option) {
+        if (option.length === 0) {
+            return '';
+        } else {
+            return option[0];
+        }
+    }
+
+    $.widget('indico.rulelistwidget', {
+        options: {
```

Comment:
actually, since you have options, i'd list them here even if they do not have a useful default value simply so they can be documented